src/OperatingSystemApp/toolbox.js

```diff
@@ -0,0 +1,19 @@
+/* exported OSToolbox */
+
+const {GObject} = imports.gi;
+const {Toolbox} = imports.toolbox;
+const {OSControlPanel} = imports.OperatingSystemApp.controlpanel;
+
+var OSToolbox = GObject.registerClass(class OSToolbox extends Toolbox {
+    _init(props = {}) {
+        super._init(props);
+        this._controlPanel = new OSControlPanel({visible: true});
+        this.add(this._controlPanel);
+        this.show_all();
+    }
+
+    bindWindow(win) {
+        this._window = win;
+    }
```

Comment:
This is not needed here.

Reply:
well bindWindow need to be declared, and you need to use the param to avoid the linter complaining. But I guess I can remove the param entirely

Reply:
Other toolboxes do something like `void (this, win)` for that